src/mds/Server.cc

```diff
@@ -6503,6 +6503,115 @@ void Server::handle_client_setvxattr(const MDRequestRef& mdr, CInode *cur)
     mdr->no_early_reply = true;
     pip = pi.inode.get();
     adjust_realm = true;
+  } else if (name == "ceph.dir.subvolume.snapdirvisibility"sv) {
```

Comment:
```suggestion
  } else if (name == "ceph.dir.subvolume.snaps.visible"sv) {
```

I could see us having other vxattrs under this "snaps" in the future. (Maybe `limit` for instance.)

Reply:
done